### ISSUE TITLE
Double Click zoom is not  zooming in with 'centerInside' scale type.

### ISSUE THREAD

**Issue by DominList** [Piasy/BigImageViewer#84]
Double Click zoom is not  zooming in with 'centerInside' value for 'initScaleType'. I want to show full image on the screen and give an ability that user can tap it twice to see the details.

**Comment by Piasy** [comment]
You mean making the tapped position become center of screen after zooming in?

Actually that's SSIV's feature, you can read its doc, or submit a issue here: https://github.com/davemorrissey/subsampling-scale-image-view

**Comment by DominList** [comment]
No. That can be another topic. I mean it doesn't work at all with initScaleType="centerInside". Reaction for image tapped twice is none or insignificant.

**Comment by Gauravv97** [comment]
Double tap to zoom is working fine for the same image in SSIV(typically the image is of height>>width such that the center fit is on the basis of height) but the image is not zooming on double tap in BigImage..

**Comment by KimiChiu** [comment]
I think it's because `SubsamplingScaleImageView.this.doubleTapZoomScale` is always 1.
So this line in the `doubleTapZoom` will always zoom to 1.
`float doubleTapZoomScale = Math.min(maxScale, 
SubsamplingScaleImageView.this.doubleTapZoomScale);`

Tried everything but still not able to change the value.
```
view.setDoubleTapZoomDpi(80);
view.setDoubleTapZoomScale(5);
view.setDoubleTapZoomDuration(200);
view.setDoubleTapZoomStyle(ZOOM_FOCUS_FIXED);
```
